conanfile.py

```diff
@@ -88,7 +90,7 @@ def requirements(self):
         self.requires("fmt/9.1.0")
         self.requires("inja/3.4.0")
         self.requires("nlohmann_json/3.11.2")
-        self.requires("incbin/cci.20211107"),
+        self.requires("incbin/cci.20211107"),   
```

Comment:
Trailing whitespace. Do you have the git hooks enabled? `make setup-git` I think.